api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -708,8 +708,8 @@ message HttpConnectionManager {
   // This affects the upstream host header.
   // Without setting this option, incoming requests with host `example.com.` will not match against
   // route with :ref:`domains<envoy_v3_api_field_config.route.v3.VirtualHost.domains>` match set to `example.com`. Defaults to `false`.
-  // When incoming requests with host/authority header has port part with host, on setting this option it removes trailing dot from host keeping port
-  // value as is (e.g. host value `example.com.:443` will be updated to `example.com:443`).
+  // When the incoming request contains a host/authority header includes a port number, setting this option will strip trailing dots
```

Comment:
`a trailing dot, if present, from the host section` 

otherwise this sounds like we strip multiple dots? I think we only do one right?

Reply:
Yes, we do remove only one dot, updated in latest commit.